### ISSUE TITLE
My plugin isn't showing up in the search 

### ISSUE THREAD

**Issue by drpepper** [parcel-bundler/parcel#9711]
# 🐛 bug report

Hi! I just realized that a plugin I made a while back doesn't show up on the [plugin browser](https://parceljs.org/plugin-browser/). 

I'm probably doing something wrong, but I'm not sure what.

The plugin is called [parcel-transformer-spritesheet](https://www.npmjs.com/package/parcel-transformer-spritesheet) 

## 🤔 Expected Behavior

I should see the plugin when I search "spritesheet" in the plugin browser.

## 😯 Current Behavior

It's not there!

Thanks for your help!

**Comment by mischnic** [comment]
It's using a Algolia npm search API: https://github.com/parcel-bundler/website/blob/v2/src/plugin-browser/hooks.js

I think that's this here: https://github.com/algolia/npm-search

So it's a black box to us as well 🤷

**Comment by drpepper** [comment]
Thanks for that! Based on the comment that it's the same search that powers [yarnpkg.com](https://yarnpkg.com/) and [jsdelivr.com](https://www.jsdelivr.com/), I tried searching there, and found my package. 

I do notice that my package has 0 downloads (I use it as a linked git module), but there's nothing that I see in the search code that would filter that out.

**Comment by mischnic** [comment]
Yeah, it's here: https://yarnpkg.com/search?q=parcel-transformer+sprite
And that is using the exact same endpoint. So one of the filters on https://github.com/parcel-bundler/website/blob/v2/src/plugin-browser/hooks.js must be the problem
